web/pgadmin/browser/static/js/heartbeat.js

```diff
@@ -26,10 +26,12 @@ export function send_heartbeat(_server_id, _item) {
         }
       })
       .catch((error) => {
-        if (error && error.message == 'Network Error') {
-          pgAdmin.Browser.notifier.error(gettext(`pgAdmin server not responding, try to login again: ${error.message || error.response.data.errormsg}`));
-        } else {
-          pgAdmin.Browser.notifier.error(gettext(`Server heartbeat logging error: ${error.message || error.response.data.errormsg}`));
+        if (!error.response?.data?.data?.includes('Manager not found')){
```

Comment:
I would just return from this if condition, rather than putting everything into this condition.

Reply:
If I directly return  then it will not call the  `stop_heartbeat(_item);`